R/R-igraphdata/Portfile

```diff
@@ -15,5 +15,3 @@ checksums           rmd160  01b953f881370ce443d0bd5b3e832238403e7532 \
 supported_archs     noarch
 
 depends_lib-append  port:R-igraph
-
-test.run            yes
```

Comment:
I would rather keep it. We would want to switch to `test.run yes` by default, eventually (well, IMO). If anything, setting `test.run yes` signals that all dependencies are in, and no additional work is needed. Also `R` allows to run checks for any package, whether special tests are included or not. Why would we prohibit that?

Reply:
Re-added. Let's merge.

Reply:
LGTM